### PR TITLE
docs: Create a guide on sourcing from Kentico Cloud

### PR DESCRIPTION
**Pull request by CollierCZ**
## Description

Added a guide on how to source content from Kentico Cloud.

### CODE REVIEW

**Comment by wardpeet** [comment]
I summon @gatsbyjs/docs & @gatsbyjs/ecosystem

File: docs/docs/sourcing-from-kentico-cloud.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+        body_copy {
+          value
+        }
+        title {
+          value
+        }
+      }
+    }
+  }
+`
```
Comment (Simply007) [inline docs/docs/sourcing-from-kentico-cloud.md]: Add an empty line for correct formatting
```suggestion
`

```

File: docs/docs/sourcing-from-kentico-cloud.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+```
+
+When you run `gatsby develop`, you'll be able to see each article as a page with content pulled from Kentico Cloud. To see a list of all pages, visit `http://localhost:8000/asdf` (or any other url that generates a 404).
```
Comment (Simply007) [inline docs/docs/sourcing-from-kentico-cloud.md]:
```suggestion
When you rerun `gatsby develop`, you'll be able to see each article as a page with content pulled from Kentico Cloud. To see a list of all pages, visit `http://localhost:8000/asdf` (or any other url that generates a 404).
```

File: docs/docs/sourcing-from-kentico-cloud.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+Now you know how to create pages programmatically and pull their content from Kentico Cloud. To get the most out of your CaaS, you'll want to also make sure your site builds automatically whenever published content changes inside Kentico Cloud.
+
```
Comment (Simply007) [inline docs/docs/sourcing-from-kentico-cloud.md]:
```suggestion

No links nor inline linked items are not resolved for the rich text element. If you want to resolve them, it is possible to use the [embedded JS SDK resolution](https://github.com/Kentico/gatsby-source-kentico-cloud#embedded-js-sdk-resolution), or it is possible to query required data in structured form for resolution and create own React components for [inline content items](https://github.com/Kentico/gatsby-source-kentico-cloud#content-items-in-rich-text-elements), [links](https://github.com/Kentico/gatsby-source-kentico-cloud#links-in-rich-text-elements), and [images](https://github.com/Kentico/gatsby-source-kentico-cloud#images-in-rich-text-elements).

```

File: docs/docs/sourcing-from-kentico-cloud.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+        });
+        resolve();
+    });
+  });
+}
+```
+
+Now create a basic template to display each article with a title and the body that you pull with a GraphQL query:
+
+````javascript:title=src/templates/article.js
```
Comment (Simply007) [inline docs/docs/sourcing-from-kentico-cloud.md]: Remove one backtick

```suggestion
```javascript:title=src/templates/article.js
```

File: docs/docs/sourcing-from-kentico-cloud.md [hunk 5]
```diff
@@ -0,0 +1,220 @@
+---
+title: Sourcing from Kentico Cloud
+---
+
+In this guide, you'll go through how to quickly source content for your Gatsby site from [Kentico Cloud](https://kenticocloud.com/).
```
Comment (Simply007) [inline docs/docs/sourcing-from-kentico-cloud.md]: Remove empty space at the end of the line.

```suggestion
In this guide, you'll go through how to quickly source content for your Gatsby site from [Kentico Cloud](https://kenticocloud.com/).
```

File: docs/docs/sourcing-from-kentico-cloud.md [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+
+You can also take advantage of Kentico Cloud's ability to deliver content in multiple languages and create relationships among various content with linked items. However you structure your content in Kentico Cloud, the [source plugin](https://github.com/Kentico/gatsby-source-kentico-cloud) will ensure the proper nodes are created for your Gatsby site.
+
+_Note: For this guide, you'll work from scratch from the [Gatsby default starter](https://github.com/gatsbyjs/gatsby-starter-default) to get a feel for how sourcing from Kentico Cloud works. If you'd like to look at an example of a complete basic site, have a look at the [Kentico Cloud Gatsby starter site](https://github.com/Kentico/gatsby-starter-kentico-cloud), which includes worked examples for querying content._
+
+## Setup
+
+### Kentico Cloud
```
Comment (Simply007) [inline docs/docs/sourcing-from-kentico-cloud.md]: Add new line
```suggestion
### Kentico Cloud

```

File: docs/docs/sourcing-from-kentico-cloud.md [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+
+And that's enough for you to be able to access content from Kentico Cloud in your site. You can see this by starting Gatsby in development mode:
+
+```shell
+gatsby develop
+```
+
+To see all the content that's available from Kentico Cloud, you can test out GraphQL queries in GraphiQL at `http://localhost:8000/___graphql`. Nodes from Kentico Cloud will be prefixed with `kenticoCloud` (for single nodes) or `allKenticoCloud` (for all examples of a given type). Then each node has `Item` or `Type`, depending on what it is, and lastly you can add the codename of a specific type you are looking for. Feel free to explore the responses in GraphiQL.
+
+To see how to put that data into your site, first go to `http://localhost:8000/`. Notice that the default title for the site is "Gatsby Default Starter". You can change that by pulling the title for your site from Kentico Cloud.
```
Comment (Simply007) [inline docs/docs/sourcing-from-kentico-cloud.md]: Remove empty line at the end of the line. 

```suggestion
To see how to put that data into your site, first go to `http://localhost:8000/`. Notice that the default title for the site is "Gatsby Default Starter". You can change that by pulling the title for your site from Kentico Cloud.
```

File: docs/docs/sourcing-from-kentico-cloud.md [hunk 8]
```diff
@@ -0,0 +1,220 @@
...
+        }
+      }
+    `}
+    render={data => (
+      <>
+        <Header siteTitle={data.kenticoCloudItemHome.elements.metadata__meta_title.value} />
+...
+```
+
+If you look at  `http://localhost:8000/`, you'll notice the title is now "Dancing Goat–Freshest coffee on the block!". You can easily change this title in Kentico Cloud to whatever you want and rerun `gatsby develop` to rebuild the site ([see below about automatic builds](#continuous-deployment)).
```
Comment (Simply007) [inline docs/docs/sourcing-from-kentico-cloud.md]: Remove space

```suggestion
If you look at `http://localhost:8000/`, you'll notice the title is now "Dancing Goat–Freshest coffee on the block!". You can easily change this title in Kentico Cloud to whatever you want and rerun `gatsby develop` to rebuild the site ([see below about automatic builds](#continuous-deployment)).
```

File: docs/docs/sourcing-from-kentico-cloud.md [hunk 9]
```diff
@@ -0,0 +1,220 @@
...
+          value
+        }
+        title {
+          value
+        }
+      }
+    }
+  }
+`
+
```
Comment (Simply007) [inline docs/docs/sourcing-from-kentico-cloud.md]: There is no need for an empty line - sorry'

```suggestion
```

**Comment by Simply007** [comment]
hello @wardpeet,

Could you please take a look at the pull request. 
This submission would not lead to failing checks from my point of view.

I have also gone through the process described in the tutorial and everything works as described.

**Comment by Simply007** [comment]
Just a ping to @gatsbyjs/docs

**Comment by calcsam** [comment]
Looks great -- I'm going to go ahead and merge this in.

**Comment by CollierCZ** [comment]
@calcsam, thanks for merging. Just a question: what would have to happen to have the guide appear in [the docs themselves](https://www.gatsbyjs.org/docs/headless-cms/) like the other sourcing guides?

**Comment by CollierCZ** [comment]
@calcsam, thanks for merging. Just a question: how would I get the guide available in the [docs themselves](https://www.gatsbyjs.org/docs/headless-cms/), like the other sourcing guides?

**Pull request merged**